sos/collector/clusters/satellite.py

```diff
@@ -0,0 +1,39 @@
+# Copyright Red Hat 2020, Jake Hunsaker <jhunsake@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from pipes import quote
+from sos.collector.clusters import Cluster
+
+
+class satellite(Cluster):
+    """Red Hat Satellite 6"""
+
+    cluster_name = 'Red Hat Satellite 6'
+    packages = ('satellite', 'satellite-installer')
+
+    def _psql_cmd(self, query):
+        _cmd = "su postgres -c %s"
+        _dbcmd = "psql foreman -c %s"
+        return _cmd % quote(_dbcmd % quote(query))
```

Comment:
This does not handle deployments with external `postgres` DB. Cf. https://github.com/sosreport/sos/blob/master/sos/report/plugins/foreman.py#L31 and below.

Reply:
Ack, will have to look into that. Do you consider this a merge blocker?

Reply:
imho not a blocker. let raise post-merge issue and I can PR it.